io/common/distro_tools.py

```diff
@@ -19,22 +19,24 @@
 """
 
 from avocado import Test
-from avocado.utils import process
+from avocado.utils import processi, distro
 from avocado.utils import pci
 from avocado.utils.software_manager.manager import SoftwareManager
 
+release = "%s%s" % (distro.detect().name, distro.detect().version)
 
 class DisrtoTool(Test):
     '''
     to test different type of tool
     '''
-
     def setUp(self):
         '''
         get all parameters
         '''
         self.option = self.params.get("test_opt", default='')
         self.tool = self.params.get("tool", default='')
+        if 'netstat' in self.tool and "SuSE15"in release:
```

Comment:
@abdhaleegit I think a space is missing here,` "SuSE15"in`
please check

Reply:
done

Reply:
done